test/index.test.js

```diff
@@ -112,7 +139,40 @@ describe('Probot', () => {
       const robot = probot.load(robot => robot.on('push', spy))
       robot.auth = jest.fn().mockReturnValue(Promise.resolve({}))
 
-      await probot.receive(event)
+      await probot.receive({
+        id: eventMeta.id,
+        name: eventMeta.name,
+        data: pushPayload,
+        signature: signature
+      })
```

Comment:
I feel like signature validation should only be a concern for events coming from untrusted sources (a.k.a. web requests). Anything calling `probot.receive` (which is currently just tests) is trusted and shouldn't have to be concerned about whether the webhook payload is signed.

Reply:
Yeah that makes sense 🤔 This is the same `.receive` method that the internal `EventHandler` module uses, which is meant to be used in server frameworks like [hapi](https://hapijs.com/) that are not compatible with the middleware provided by `@octokit/webhooks`. 

I’ll give this some more though